src/sync/discovery.ts

```diff
@@ -3,57 +3,97 @@ import { getRepositorySummary } from "./jobs";
 import enhanceOctokit from "../config/enhance-octokit";
 import { Application } from "probot";
 import { Repositories, SyncStatus } from "../models/subscription";
-import {LoggerWithTarget} from "probot/lib/wrap-logger";
+import { LoggerWithTarget } from "probot/lib/wrap-logger";
 import { sqsQueues } from "../sqs/queues";
+import GitHubClient from "../github/client/github-client";
+import { getCloudInstallationId } from "../github/client/installation-id";
+import { Repository } from "@octokit/graphql-schema";
+import { booleanFlag, BooleanFlags } from "../config/feature-flags";
 
 export const DISCOVERY_LOGGER_NAME = "sync.discovery";
+const MAX_PAGE_SIZE_REPOSITORY = 100;
+
+const getAllRepositories = async (github): Promise<Repository[]> => {
```

Comment:
Alternatively, I could loop and fire off a queue event each 100 repos, I was just a bit hesitant to make too much change, since the goal is just replacing the client...... risk vs value kinda thing.

Reply:
We definitely should separate it by different queue messages (Keep cursor in DB and schedule a new SQS message for each page).

However for now we are just changing the client. We definitely need a ticket for that (If we don't have it already)